test/dx_mgn_pool_simulation.js

```diff
@@ -0,0 +1,150 @@
+const MintableERC20 = artifacts.require("ERC20Mintable.sol")
+const EtherToken = artifacts.require("EtherToken")
+const TokenFRT = artifacts.require("TokenFRT")
+
+const DX = artifacts.require("DutchExchange")
+const DXProxy = artifacts.require("DutchExchangeProxy")
+const Coordinator = artifacts.require("Coordinator")
+const DxMgnPool = artifacts.require("DxMgnPool")
+
+const { waitUntilPriceIsXPercentOfPreviousPrice } = require("./dx_e2e")
```

Comment:
maybe it would be a little bit cleaner if we move this function waitUntil.... into utiliities